test/karma/karma.config.js

```diff
@@ -94,16 +98,21 @@ module.exports = function (config) {
     urlRoot: '/__karma__/',
     files: [
       // 'test-app/prerender-test/karma.spec.ts',
-      'test-app/**/*.spec.ts',
-      'test-app/util.ts',
-      'test-app/assets/angular.min.js',
-      { pattern: 'www/**/*', watched: false, included: false, served: true, nocache: true, type: 'module' },
+      'test-app/**/*.spec.ts', // tells karma these are tests we need to serve & run
+      'test-app/util.ts', // used by 'www' output target tests to load components
+      'test-app/assets/angular.min.js', // used by a 'www' output target test
+      {
+        pattern: path.join(WWW_OUT_DIR, '/**/*'),
```

Comment:
Only this value changed, but because I pushed it over the line length limit, we see some churn 😢